### PR TITLE
Add Saint Patrick's day folder with banner and server icon

### PR DESCRIPTION
**Pull request by MrKomodoDragon**
I added the assets for st. patrick’s day, which are a banner and server icon. Please let me know if I need to move the files around to different folders. Thank you!

### CODE REVIEW

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
Hi! Thanks for the PR!  This looks great, but there are a few things I'd like to mention.

Seems like there's some weird blue on the banner logo
Screenshot 2021-02-22 at 21 09 52

There's also some weird outlines on the server icon:
Screenshot 2021-02-22 at 21 11 25
Screenshot 2021-02-22 at 21 12 17

It seems like this was done using something like Photoshop. I'd recommend Illustrator, Affinity Designer or Inkscape, so you can work directly with the vector shapes. (Ping `Vestergurkan` on the server if you'd like me to explain more).

Considering that the server icon will be tiny, I'm fine with some minor defects in the edges, but if you could go over the edges and see if it can be fixed, that would be awesome! If you are in fact using Photoshop, you can also try out so called "adjustment layers" for modifying only the colors.

Just fix the blue in the banner and I'll be happy this time. Thanks a lot!

**Comment by MarkKoz** [comment]
If all you need to do is change colours, it's really simple to open the SVG as text and edit the colours. It's very similar to HTML/CSS.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
Great to see that you're doing it with Illustrator instead now! A few things:
- Seems like there's one `.DS_Store` left. Could you add that to `.gitignore` so we won't have issues with it being committed in the future?
- It seems like the banner has some weird artifacting. Do you know what might be up? Check it out below:
  Screenshot 2021-02-23 at 21 48 42

Other than that, it's looking pretty good.

**Comment by MrKomodoDragon** [comment]
I dunno about the artifacting, but will put the .DS_store in .gitignore.

**Comment by gustavwilliam** [comment]
The banner was previously an sag that only contained an image, which led to the strange artifacting of the fill tool. After discussing this in #media-branding, I've recreated the banner using the colors and dimensions of the original file. Since it seems like an svg can't be uploaded in a GitHub comment, I'll send the file to @MrKomodoDragon over on Discord and he'll export it as a png.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+seasonal/evergreen/.DS_Store
```
Comment (gustavwilliam) [inline .gitignore:1]:
```suggestion
.DS_Store
```
This will make it apply to all `.DS_Store`s, instead of just the ones in the specified folder.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
One final thing I'm gonna comment on, unless something changes. The file `events/st_patricks_day/st_patricks_banner (1).svg` should be renamed to `events/st_patricks_day/st_patricks_banner.svg`. Other than that, I've left a single comment for you to check out.

**Review by gustavwilliam — APPROVED** [review]
LGTM, thanks! Something seems to make the banner svg very tall, but I don't know what this is about. The shapes are right and it looks (almost) completely right in Illustrator, so I'll approve it. However, if someone knows what could be done about it, I'd love to hear about it.

**Comment by MarkKoz** [comment]
The banner has a white border all around:

banner

Presumably this is a mistake since none of the other banners are like this. This is what it should be:

banner

**Comment by MarkKoz** [comment]
Oh, it seems like exporting from the SVG results in a border but the PNG in this PR looks fine. So only the SVG is messed up.

**Comment by gustavwilliam** [comment]
@MarkKoz yeah. After digging into this, it seems like the viewbox for the svg was messed up. We fixed it by creating a new artboard in Illustrator and pasting the contents on it. Pretty weird, but it seems to be working now.

**Review by lemonsaurus — APPROVED** [review]
Looks good!

**Comment by lemonsaurus** [comment]
Thanks @gustavwilliam for the thorough review. Nothing left for me to do here but merge.

**Pull request merged**